src/App.jsx

```diff
@@ -16,7 +17,11 @@ export default function App() {
     <ThemeProvider theme={theme}>
       <StepContext.Provider value={{ totalStep: TOTAL_STEP, currentStep, setCurrentStep }}>
         <FooterContext.Provider value={{ activeNext, setActiveNext }}>
-          <Step>{currentStep.number && step[currentStep.number - 1]}</Step>
+          {!currentStep.number ? (
+            <FirstPage />
+          ) : (
+            <Step>{currentStep.number && step[currentStep.number - 1]}</Step>
```

Comment:
```suggestion
            <Step>{step[currentStep.number - 1]}</Step>
```
위에서 currentStep.number  값을 검증했기 때문에 추가 검증은 필요없을 것 같습니다!

Reply:
수정해서 올려두었습니다!